src/lang/english.txt

```diff
@@ -380,21 +380,21 @@ STR_TOOLBAR_TOOLTIP_DISPLAY_STORY_BOOK                          :{BLACK}Display
 STR_TOOLBAR_TOOLTIP_DISPLAY_GOALS_LIST                          :{BLACK}Display goal list
 STR_TOOLBAR_TOOLTIP_DISPLAY_GRAPHS                              :{BLACK}Display company graphs and cargo payment rates
 STR_TOOLBAR_TOOLTIP_DISPLAY_COMPANY_LEAGUE                      :{BLACK}Display company league table
-STR_TOOLBAR_TOOLTIP_FUND_CONSTRUCTION_OF_NEW                    :{BLACK}Examine industries or fund construction of a new industry
+STR_TOOLBAR_TOOLTIP_FUND_CONSTRUCTION_OF_NEW                    :{BLACK}Display industry directory
```

Comment:
`STR_TOOLBAR_TOOLTIP_DISPLAY_MAP`, `STR_TOOLBAR_TOOLTIP_SHOW_LAST_MESSAGE_NEWS` and `STR_TOOLBAR_TOOLTIP_LAND_BLOCK_INFORMATION` essentially list all options. Why should this tooltip just show one of the three options?

Reply:
Ah, maybe the right answer is to make that the standard here. 😄